default.nix

```diff
@@ -0,0 +1,13 @@
+{ stdenv, pkgs }:
+
+stdenv.mkDerivation {
+    name = "breakfastbot";
+    propagatedBuildInputs = [
```

Comment:
You shouldn't need propagatedBuildInputs here, just buildInputs should be fine. 
`stdenv` has a hook to automatically replace the reference to python in the shebang with an absolute path to the python env available at build time (patchShebangs)